thirdparty/github.com/go-yaml/yaml.v3/yamlh.go

```diff
@@ -1,807 +1,707 @@
 //
 // Copyright (c) 2011-2019 Canonical Ltd
-// Copyright (c) 2006-2010 Kirill Simonov
```

Comment:
same question here… I expected this commit to have minimal changes over the yaml.v3 copy